integration/aggregate_compat_test.go

```diff
@@ -678,8 +678,8 @@ func TestAggregateCompatGroup(t *testing.T) {
 				bson.D{{"$sort", bson.D{{"_id", 1}}}},
 				bson.D{{"$group", bson.D{
 					{"_id", bson.D{
-						{"missing1", "$non-existent1"},
-						{"missing2", "$non-existent2"},
+						{"missing1", "$nonexistent1"},
+						{"missing2", "$nonexistent2"},
```

Comment:
I'm not quite sure what the second arg here is doing/how it's being parsed, but in general it's unlikely that my change is particularly harmful.